Source/Core/Core/HW/Memmap.cpp

```diff
@@ -57,6 +60,71 @@ u8* m_pL1Cache;
 u8* m_pEXRAM;
 u8* m_pFakeVMEM;
 
+static u32 get_MEM1Size()
+{
+  u32 MEM1_Size;
+  // I need to initialize some stuff that is normally initialized in
+  // int main(), because this funtion will be run before any of that.
+  UICommon::SetUserDirectory("");
```

Comment:
This is kind of a large code smell. I don't think we should be calling UI specifics from Core.